velox/functions/lib/window/NthValue.cpp

```diff
@@ -32,34 +32,35 @@ class NthValueFunction : public exec::WindowFunction {
       : WindowFunction(resultType, pool, nullptr), ignoreNulls_(ignoreNulls) {
     VELOX_CHECK_EQ(args.size(), 2);
     VELOX_CHECK_NULL(args[0].constantValue);
+    auto offsetType = args[1].type;
+    VELOX_CHECK(
```

Comment:
USER_CHECK

Reply:
Thanks for the suggestion, but I don't have access to this upstream branch anymore. Would it be fine if this change is made in a subsequent PR?

Reply:
@pramodsatya Can you maybe create a new PR and close this one? Use gh pr checkout 6186.

Reply:
Sure, opened #8054 and closing this, thanks.